docs/zkapps/tutorials/09-recursion.mdx

```diff
@@ -66,7 +66,7 @@ To create a ZkProgram, start with the `init()` method.
 ```typescript
 const Add = ZkProgram({
   name: 'add-example',
-  publicInputType: Field,
+  publicInput: Field,
```

Comment:
similar feedback 

this fix is outside the scope of this PR

can you remove this file update from the PR?
One way to get this file from the main branch is:

`git checkout main docs/zkapps/o1js/recursion.mdx`